types/stylelint/index.d.ts

```diff
@@ -282,6 +282,7 @@ declare namespace stylelint {
 		>;
 		'annotation-no-unknown': CoreRule<true, { ignoreAnnotations: OneOrMany<StringOrRegex> }>;
 		'at-rule-allowed-list': CoreRule<OneOrMany<string>>;
+		'at-rule-descriptor-value-no-unknown': CoreRule<true>;
```

Comment:
**If** #8239 gets merged before this one

```suggestion
		'at-rule-descriptor-value-no-unknown': CoreRule<
			true,
			{},
			{ rejected: (descriptor: string, value: string) => string }
		>;
```

The third argument is optional so it's not mandatory though.